media-proxy/src/mesh/st2110_20tx.cc

```diff
@@ -0,0 +1,65 @@
+#include "st2110tx.h"
+
+namespace mesh::connection {
+
+st_frame *ST2110_20Tx::get_frame(st20p_tx_handle h)
+{
+    return st20p_tx_get_frame(h);
+};
+
+int ST2110_20Tx::put_frame(st20p_tx_handle h, st_frame *f)
+{
+    return st20p_tx_put_frame(h, f);
+};
+
+st20p_tx_handle ST2110_20Tx::create_session(mtl_handle h, st20p_tx_ops *o)
+{
+    return st20p_tx_create(h, o);
+};
+
+int ST2110_20Tx::close_session(st20p_tx_handle h)
+{
+    return st20p_tx_free(h);
+};
+
+Result ST2110_20Tx::configure(context::Context &ctx, const std::string &dev_port,
+                              const MeshConfig_ST2110 &cfg_st2110,
+                              const MeshConfig_Video &cfg_video)
+{
+    if (cfg_st2110.transport != MESH_CONN_TRANSPORT_ST2110_20) {
+        set_state(ctx, State::not_configured);
+        return set_result(Result::error_bad_argument);
+    }
+
+    if (configure_common(ctx, dev_port, cfg_st2110))
+        return set_result(Result::error_bad_argument);
```

Comment:
State will stay `configuring` here.

Reply:
Fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/261/commits/4db69d1907a638e19df895bab4b5914102226a85